src/ROUTES.js

```diff
@@ -17,7 +17,10 @@ export default {
     IOU_BILL: 'iou/split',
     SEARCH: 'search',
     SIGNIN: 'signin',
-    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:validateCode',
+    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:accountID/:validateCode',
+    PROFILE: 'profile',
+    PROFILE_WITH_LOGIN: 'profile/:login',
+    getProfileRoute: login => `profile/${login}`,
```

Comment:
These changes don't look like they should be here.

```js
    PROFILE: 'profile',
    PROFILE_WITH_LOGIN: 'profile/:login',
    getProfileRoute: login => `profile/${login}`,
```

I think `profile` was changed to `details` recently so that's not super obvious but will need to be fixed.